src/app/workflow/workflow.component.ts

```diff
@@ -429,10 +433,16 @@ export class WorkflowComponent extends Entry implements AfterViewInit, OnInit {
   onSelectedVersionChange(version: WorkflowVersion): void {
     this.selectedVersion = version;
     if (this.selectVersion) {
-      const prefix = this.entryType === EntryType.BioWorkflow ? ga4ghWorkflowIdPrefix : ga4ghServiceIdPrefix;
-      this.gA4GHFilesService.updateFiles(prefix + this.workflow.full_workflow_path, this.selectedVersion.name, [
-        this.descriptorTypeCompatService.stringToDescriptorType(this.workflow.descriptorType),
-      ]);
+      if (this.entryType === EntryType.AppTool) {
+        this.gA4GHFilesService.updateFiles(this.workflow.full_workflow_path, this.selectedVersion.name, [
+          this.descriptorTypeCompatService.stringToDescriptorType(this.workflow.descriptorType),
+        ]);
+      } else {
+        let prefix = this.entryType === EntryType.BioWorkflow ? ga4ghWorkflowIdPrefix : ga4ghServiceIdPrefix;
+        this.gA4GHFilesService.updateFiles(prefix + this.workflow.full_workflow_path, this.selectedVersion.name, [
```

Comment:
This a small thing, but the only thing different between this block and line 437 is the first param, so I would only have `updateFiles` once in the code; it makes it easier to read.

The if/else could just set up the first parameter, either as is, or in a method.